### ISSUE TITLE
[FR] can you make a new unbound + pihole based on Alpine?

### ISSUE THREAD

**Issue by r10513** [madnuttah/unbound-docker#4]
Hi,
I like the fact that your unbound image is quite tiny (28,9MB), but, if I use your compose.yaml file to deploy also pi-hole, there are two issues:

1. The two images are independent (so, two IP addresses)
2. The pihole image is based on the latest pihole - while this is good, it means that this image is 301,3MB.

Would it be possible to create a new docker image that uses your unbound as a prerequisite, but deploys pihole inside that unbound?
There is something similar already done (cbcrowe/pihole-unbound:latest), but based on Debian, and not on Alpine (and its size is 342,6MB).

Thanks!

**Comment by madnuttah** [comment]
I'll check that, thank you for suggesting.

**Comment by r10513** [comment]
Hi, if it helps, you can borrow some ideas from https://www.reddit.com/r/linux/comments/ovuzcm/pihole_for_alpine_linux_x86_and_x86_64/

I have not tested it yet (maybe tomorrow I will give it a try on a plain vanilla Alpine Linux).

Thank you very much for looking into it.

**Comment by madnuttah** [comment]
Hi, 

sorry for my late reply and that I may disappoint you. I've checked it by trying to install it on a plain Alpine Image and there's too much effort needed to make it work and even if I released it, I wouldn't have the time to maintain install scripts, workarounds and such. There are some hints around the web, I guess you found them already by yourself and even if it will start it runs quite unstable. I just don't want to spend much more time into a Kobayashi Maru mission. May one day the Pi-hole devs make an Alpine compatible version. Until then, your best bet is to use the official image which does a great job running perfectly stable.
